[
    "Match found, the package 'libpython3.10-stdlib', is vulnerable to 'CVE-2022-42919'. Current version: '3.10.4-3' (less than '3.10.6-1~22.04.1' or equal to ''). - Agent '' (ID: '002', Version: '').",
    "Match found, the package 'libpython3.10-stdlib', is vulnerable to 'CVE-2022-45061'. Current version: '3.10.4-3' (less than '3.10.6-1~22.04.2' or equal to ''). - Agent '' (ID: '002', Version: '').",
    "Match found, the package 'libpython3.10-stdlib', is vulnerable to 'CVE-2023-24329'. Current version: '3.10.4-3' (less than '3.10.6-1~22.04.2ubuntu1' or equal to ''). - Agent '' (ID: '002', Version: '').",
    "No match due to default status for Package: libpython3.10-stdlib, Version: 3.10.4-3 while scanning for Vulnerability: CVE-2023-41105"
]
